image2tensor/src/image2tensor.rs

```diff
@@ -14,6 +14,24 @@ pub enum ColorOrder {
     BGR,
 }
 
+pub fn convert_dynamic_image_bytes_to_tensor_bytes(
+    image: DynamicImage,
```

Comment:
I fixed the typo in this function but are we sure this function should be `pub`? It reexports `DynamicImage` from the `image` crate which is a bit weird; maybe you meant to just factor this logic out but leave it private?

Reply:
Not pub is fine. The only time where using a DynamicImage directly would happen is if you did some preprocessing on the image before converting it to a Tensor. In that case, it would save the time of rendering to bytes and reading back out.

Reply:
Cool, I rebased this and added that change.